apps/src/templates/instructions/RubricField.jsx

```diff
@@ -72,9 +72,32 @@ class RubricField extends Component {
     rubricValue: PropTypes.string.isRequired,
     disabledMode: PropTypes.bool,
     onChange: PropTypes.func.isRequired,
-    currentlyChecked: PropTypes.bool
+    currentlyChecked: PropTypes.bool,
+    expandByDefault: PropTypes.bool
   };
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      detailsOpen: false
+    };
+  }
+
+  componentDidMount = () => {
+    const details = document.querySelector(
+      `#rubric-details-${this.props.rubricLevel}`
```

Comment:
this doesn't look right, because this will only work the first time you render the component. to get a reference to an html element that works after every time the component renders, you probably want to use a ref: https://reactjs.org/docs/refs-and-the-dom.html